common.py

```diff
@@ -112,6 +112,21 @@ def parse_command_line_args():
         default=False,
         choices=['on', 'off']
     )
+    parser.add_argument(
+        "--on-archived",
+        help="Tells the tool what to do when a GitHub project is archived (Snyk projects retained by default)",
+        required=False,
+        default="retain",
+        choices=['retain', 'deactivate', 'delete']
```

Comment:
let's use the same language when no changes are to be made, e.g. ignore to match --on-unarchived, or nothing as suggested by @gwnlng